plugins/modules/pacemaker_resource.py

```diff
@@ -0,0 +1,298 @@
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+options:
+  state:
+    description:
+      - Indicate desired state for cluster resource.
+    choices: [ create, delete ]
+    required: true
+    type: str
+  name:
+    description:
+      - Specify the resource name to create.
+    aliases: [ resource_id ]
+    required: true
+    type: str
+  resource_type:
+    description:
+      - Resource type to create.
+    aliases: [ type ]
+    type: dict
+    suboptions:
+      resource_name:
+        description:
+          - Specify the resource type name.
+        type: str
+      resource_standard:
+        description:
+          - Specify the resource type standard.
+        type: str
+      resource_provider:
+        description:
+          - Specify the resource type providers.
+        type: str
+  resource_option:
+    description:
+      - Specify the resource option to create.
+    aliases: [ option ]
+    type: list
+    elements: str
+    default: []
+  resource_operation:
+    description:
+      - List of operations to associate with resource.
+    aliases: [ op ]
+    type: list
+    elements: dict
+    default: []
+    suboptions:
+      operation_action:
+        description:
+          - Operation action to associate with resource.
+        type: str
+      operation_option:
+        description:
+          - Operation option to associate with action.
+        type: list
+        elements: str
+  resource_meta:
+    description:
+      - List of meta to associate with resource.
+    type: list
+    elements: str
+  resource_argument:
+    description:
+      - Actions to associate with resource.
+    type: dict
+    suboptions:
+      argument_action:
+        description:
+          - Action to apply to resource.
+        type: str
+        choices: [ clone, master, group, promotable ]
+      argument_option:
+        description:
+          - Options to associate with resource action.
+        type: list
+        elements: str
+  disabled:
+    description:
+      - Specify argument to disable resource.
+    type: bool
+    default: false
+  wait:
+    description:
+      - Timeout period for polling the resource creation.
+    type: int
+    default: 300
+'''
+
+EXAMPLES = '''
+---
+- name: Create pacemaker resource
+  hosts: localhost
+  gather_facts: false
+  tasks:
+  - name: Create virtual-ip resource
+    community.general.pacemaker_resource:
+      state: create
+      name: virtual-ip
+      resource_type:
+        resource_name: IPaddr2
+      resource_option:
+        - "ip=[192.168.1.2]"
+      resource_argument:
+        argument_action: group
+        argument_option:
+          - master
+      resource_operation:
+        - operation_action: monitor
+          operation_option:
+            - interval=20
+'''
+
+RETURN = '''
+cluster_resources:
+    description: The cluster resource output message.
+    type: str
+    sample: '"Assumed agent name ocf:heartbeat:IPaddr2 (deduced from IPaddr2)"'
```

Comment:
I believe the double quoting here is not necessary. This:
```suggestion
    sample: "Assumed agent name ocf:heartbeat:IPaddr2 (deduced from IPaddr2)"
```
should do the trick.